keyboards/kbdfans/kbd8x_mk2/readme.md

```diff
@@ -0,0 +1,15 @@
+# KBD8X MKII
+
+![Photo](https://cdn.shopify.com/s/files/1/1473/3902/products/9_e4262f46-0d69-4c6c-9cc6-b88a3e2483d7_1800x1800.jpg)
+
+A fairly typical TKL
```

Comment:
lol fairly typical

Reply:
also, it doesn't quite match the `tkl_ansi` layout we have.  :(